test/randomized/test_synthesis.py

```diff
@@ -64,21 +63,20 @@ def test_exact_supercontrolled_decompose_random(self, seeds):
         decomposer = TwoQubitBasisDecomposer(UnitaryGate(basis_unitary))
         self.check_exact_decomposition(random_unitary(4, seed=seeds[4]).data, decomposer)
 
-    @given(strategies.tuples(*[rotation] * 6), seed)
```

Comment:
What did this `seed` do? 🙂

Reply:
It was used as a seed to the unitary simulator, which is now no longer used (I don't think `Operator` takes a seed).